.dev/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:12.6-alpine
+
+RUN apk add --update --no-cache bash make
+RUN npm install npm install katacoda-cli -g
```

Comment:
Could we clean up the double npm install please?

Reply:
Oh thx!
First I included there also [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli) but decided that this should be not part of this PR.
Did not clean up all of the stuff :man_facepalming: (should not hurry.. :) )

- [ ] cleanup